flutter/ios/sentry_flutter.podspec

```diff
@@ -11,7 +11,7 @@ Sentry SDK for Flutter with support to native through sentry-cocoa.
   s.source           = { :git => "https://github.com/getsentry/sentry-dart.git",
                          :tag => s.version.to_s }
   s.source_files = 'Classes/**/*'
-  s.dependency 'Sentry', '~> 6.1.4'
+  s.dependency 'Sentry', '~> 6.2.1'
```

Comment:
6.1.4 didn't have `sendDefaultPii`

Reply:
update our samples too please

Reply:
eg https://github.com/getsentry/sentry-dart/pull/278/files
or we remove the lockfile from the repo, so it updates every time, wdyt?

Reply:
The Podfile.lock is already gitignored: https://github.com/getsentry/sentry-dart/blob/main/.gitignore
I would remove it. Seems easier to manage.

Reply:
sounds good to me, I guess I've removed a few other lock files and this one was missing

Reply:
It's already removed https://github.com/getsentry/sentry-dart/commit/b1a68173134c9e2335786c37590ff4aa64a9a546

Reply:
It works with a clean repo. Just set it up from zero yesterday and I didn't have any problems.

Reply:
@ueman but you cloned the repo that had the lock file, that's what I meant, but that's what CI does pretty much, so we are good